docs/topics/making-open-source-more-inclusive.adoc

```diff
@@ -2,11 +2,13 @@
 //
```

Comment:
I didn't realize you were doing this. No need to add assembly lines to this module. It's like common-attributes. It's everywhere, so you do not need to update the assemblies.